api/tests/Behat/bootstrap/v2/Common/FormFillingTrait.php

```diff
@@ -0,0 +1,304 @@
+<?php
```

Comment:
Nice job on this 👏🏻 will definitely save a lot of time when writing these tests now not having to duplicate form fields